### PR TITLE
Extend basic css functionality to scss files

### PR DESCRIPTION
**Pull request by SandeepJoel**

### CODE REVIEW

File: code/code.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
     # 'cplusplus': 'cpp hpp',
     "csharp": "cs",
     "css": "css",
+    "scss": "scss",
```
Comment (Will-Sommers) [inline code/code.py:18]: Can you delete `sass` on line 39 below? It also looks like this is a sorted list so we may want to move this down.

**Review by Will-Sommers — COMMENTED** [review]
Looks pretty good!

**Review by auscompgeek — APPROVED** [review]
Nice.

**Review by splondike — APPROVED** [review]

**Comment by splondike** [comment]
Thanks!

**Pull request merged**